javascript/statements.json

```diff
@@ -1563,11 +1563,7 @@
               "version_added": "61"
             },
             "edge": {
-              "version_added": "15",
```

Comment:
We should keep the old value (and put the new one in front). Something like (indentation incorrect):
            {
              "version_added": "16"
            },
            {
               "version_added": "15",
              "flag": {
                "name": "Experimental JavaScript Features",
                "type": "preference"
              }